MAUI/Popup/working-with-mvvm.md

```diff
@@ -0,0 +1,444 @@
+---
+layout: post
+title: MVVM in .NET MAUI Popup control | Syncfusion
+description: Learn here all about MVVM support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.

```

Comment:
description: Learn here all about MVVM support in the Syncfusion .NET MAUI Popup (SfPopup) control, its elements, and more.